stable/mariadb/templates/NOTES.txt

```diff
@@ -0,0 +1,17 @@
+Getting Started:
+
+MariaDB can be accessed via port 3306 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+To connect to your database:
```

Comment:
I started working on this (but got distracted!), I was using a single command to run the client though, which I think will be easier:
```
kubectl run {{ template "fullname" . }}-client --rm --tty -i --image bitnami/mariadb --command -- mysql -h {{ template "fullname" . }} {{- if .Values.mariadbRootPassword }} -p {{ .Values.mariadbRootPassword }}{{- end -}}
```

What do you think?